Automattic/jetpack
# Social Notes: Add the featured image block

Fixes https://github.com/Automattic/jetpack-reach/issues/244

## Proposed changes:

* Added the featured image block to the social notes template 0ee43cc59ea7ab7a6be5773b2579fc65b88b6bb0
* Hid the media picker for notes 06b9ff8ef25acb2a09abb9f62a6b73982dbca0d5

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/244
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Create a new Social Note. You should see the paragraph and the featured image block below it.
* In the Jetpack sidebar you should see the share as a social post toggle. It should be disabled.
* Add a featured image in the block, the toggle should become active. Check the shares.
* Do the same with adding the image through the post sidebar.

![CleanShot 2024-03-28 at 20 39 21 png](https://github.com/Automattic/jetpack/assets/36671565/5a58f0fb-3edf-493a-9dd1-56e87f559c2a)
